### PR TITLE
[DO NOT MERGE] Logging/remove cursors

### PR DESCRIPTION
**Pull request by Gerhut**

### CODE REVIEW

File: src/utils/JobLogUtils.py [hunk 1]
```diff
@@ -5,30 +5,31 @@
 
 from config import config
 
-def GetJobLog(jobId, cursor=None, size=None):
+def GetJobLog(jobId, size=10000):
```
Comment (Anbang-Hu) [inline src/utils/JobLogUtils.py]: if more than 10000 lines of logs are generated between two get job log requests, then we are missing logs?

File: src/ClusterManager/joblog_manager.py [hunk 2]
```diff
@@ -40,42 +41,23 @@ def extract_job_log(jobId, logPath, userId):
...
-        for (container_id, log_text) in container_logs.items():
+        for containerId, containerLogLines in itertools.groupby(
+            lines,
+            key=lambda line: line["docker"]["container_id"]
+        ):
             try:
-                containerLogPath = os.path.join(jobLogDir, "log-conatainer-" + container_id + ".txt")
-                with open(containerLogPath, 'a') as f:
-                    f.write(log_text)
+                containerLogPath = os.path.join(jobLogDir, "log-conatainer-" + containerId + ".txt")
```
Comment (Anbang-Hu) [inline src/ClusterManager/joblog_manager.py]: @xudifsd if we decide to remove `/job`, do we still want write logs to files?

Comment (xudifsd) [reply src/ClusterManager/joblog_manager.py]: No, we should remove this dump to NFS

File: src/ClusterManager/test/utils.py [hunk 3]
```diff
@@ -276,31 +276,22 @@ def block_until_state_in(rest_url, jid, states, timeout=300):
...
+    args = {
+        "userName": email,
+        "jobId": jid,
+    }
+    args = urllib.parse.urlencode(args)
+    url = urllib.parse.urljoin(rest_url, "/GetJobLog") + "?" + args
+    resp = requests.get(url)
+    resp_json = resp.json()
+    log = resp_json["log"]
+    logger.info("%s, %s", jid, repr(log)[:10])
```
Comment (xudifsd) [inline src/ClusterManager/test/utils.py]: maybe use logger.debug

File: src/utils/JobLogUtils.py (hunk 1 above)
Comment (Gerhut) [reply src/utils/JobLogUtils.py]: Resolved via elasticsearch scroll/scan, no more 10000 lines limit

File: src/ClusterManager/test/utils.py (hunk 3 above)
Comment (Gerhut) [reply src/ClusterManager/test/utils.py]: Removed

File: src/ClusterManager/joblog_manager.py (hunk 2 above)
Comment (xudifsd) [reply src/ClusterManager/joblog_manager.py]: @Gerhut  Still didn't removed this?

Comment (Gerhut) [reply src/ClusterManager/joblog_manager.py]: Not sure your plan, should I rebase dltsdev branch?

Comment (xudifsd) [reply src/ClusterManager/joblog_manager.py]: Sorry, missed this. You can just remove this logic in both ES clusters and non-ES clusters. We are trying to remove dependency on NFS.

**Pull request closed without merging**